frontend/src/components/transactions/ActionsList.tsx

```diff
@@ -1,31 +1,220 @@
 import React from "react";
+import { Row, Col } from "react-bootstrap";
 import * as T from "../../libraries/explorer-wamp/transactions";
 
 import ActionRow, { ViewMode } from "./ActionRow";
+import ActionRowBatch from "./ActionRowBatch";
+
+import AccountLink from "../utils/AccountLink";
+import TransactionLink from "../utils/TransactionLink";
+import Timer from "../utils/Timer";
 
 export interface Props {
   actions: (T.Action | keyof T.Action)[];
   transaction: T.Transaction;
   viewMode?: ViewMode;
   reversed?: boolean;
+  TD?: boolean;
```

Comment:
What is this?

Reply:
on TransactionDetail page, kendall think no more need to show detail on each transaction row, because it is already show on the title, so I add this to check whether on the detail page to just show transaction information

Reply:
1. TD is completely broken naming. Please, never use such names. I believe, we can completely avoid this special case here.
2. Merge `ActionRowBatch` into `ActionRow` (extend `viewMode` if necessary). Please, avoid code duplication.